packages/openapi3/src/cli/actions/convert/utils/decorators.ts

```diff
@@ -29,7 +29,7 @@ export function getExtensions(element: Extensions): TypeSpecDecorator[] {
   return decorators;
 }
 function normalizeObjectValue(source: unknown): string | number | object | TSValue {
-  if (typeof source === "object") {
+  if (source !== null && typeof source === "object") {
```

Comment:
would doing the check the opposite allow removing the `as object` undeneath?

Reply:
I see, thanks for clarifying. Updated.